### PR TITLE
Add hubspot stats for pecan

### PR DESCRIPTION
**Pull request by javierarce**
Sends stats to hubspot:

* When the user open the pecan list.
* When the user clicks on a preview.

### CODE REVIEW

**Comment by javierarce** [comment]
cc @matallo & @javisantana

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5177/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5179/))

**Comment by matallo** [comment]
:+1: I can help you with app_config tomorrow, sorry I had to run

**Comment by javierarce** [comment]
Oh, no problem, Santi help me out with the config. Thanks for the help with hubspot!

**Comment by javierarce** [comment]
retest this please

**Comment by matallo** [comment]
:bow:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5181/))

File: lib/assets/javascripts/cartodb/old_common/metrics.js [hunk 1]
```diff
@@ -82,6 +82,12 @@
           case 'visited_dashboard_first_time':
             event_id = window.hubspot_ids.visited_dashboard_first_time
             break;
+          case 'applied_pecan':
+            event_id = window.hubspot_ids.applied_pecan
```
Comment (saleiva) [inline lib/assets/javascripts/cartodb/old_common/metrics.js]: Are we storing the type of map? Choropleth? Category? Boolean? The ramp?

Comment (javierarce) [inline lib/assets/javascripts/cartodb/old_common/metrics.js]: No, this ticket is just to send simple usage stats (which is quicker to implement and doesn't require touch the backend)

We are discussing the collection of more detailed stats here: https://github.com/CartoDB/cartodb/issues/4276

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5187/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5189/))

**Pull request merged**